packages/runtime/src/webcontainer/terminal-config.ts

```diff
@@ -67,6 +67,7 @@ export class TerminalPanel implements ITerminal {
   private _terminal?: ITerminal;
   private _process?: WebContainerProcess;
   private _data: string[] = [];
+  private _input: string[] = [];
```

Comment:
To be correct we should have a single array of `{ data: string, type: 'input' | 'echo' }`. That way we preserve the order of both (otherwise we will get them in out of order).

Reply:
Good catch, I'll add that!